### PR TITLE
refactor(auth): add single jwt token with service worker

### PR DESCRIPTION
**Pull request by Mackbellemore**
Remove the cookies since they arent working for now.

### CODE REVIEW

File: client/src/pages/_app.js [hunk 1]
```diff
@@ -1,12 +1,33 @@
...
+  useEffect(() => {
+    if ('serviceWorker' in navigator) {
+      window.addEventListener('load', async () => {
+        try {
+          await navigator.serviceWorker.register('/authServiceWorker.js');
+        } catch {
+          toast({
+            position: 'bottom',
+            title: 'An error occurred.',
+            description: 'Please contact Gordon',
```
Comment (Mackbellemore) [inline client/src/pages/_app.js:21]: Im not removing this

Comment (gordonpn) [reply client/src/pages/_app.js:21]: LMAO

**Review by gordonpn — APPROVED** [review]
Aight let's see this in develop

File: client/src/pages/_app.js (hunk 1 above)
Comment (SiuYe) [reply client/src/pages/_app.js:21]: nice!

**Review by SiuYe — APPROVED** [review]

**Pull request merged**